### ISSUE TITLE
The visual cue for Lopping being active is too subtle

### ISSUE THREAD

**Issue by petersampsonaudacity** [audacity/audacity#2796]
**Describe the bug**
The visual cue for Looping being active is too subtle - and thus easy to overlook.

The Loop button is actually a state-change for the Pay and Play-at-Speed buttons it doe _**NOT**_ of itself effect playback.

It is possible to have  a Loop play region and a selection both extant and pressing Play (or Plat-at-Speed) will give different behaviors depending on whether the Loop button is down or up/

**To Reproduce**
1. Generate a 30 second chirp
2. create a Loop region from 15-20 seconds
3. Observe: Loop button is automatically depressed by the making of the Loop region - but the visual cue is subtle and easily overlooked
4. Make a selection from 5-10 seconds
5. Press Play
6. Observe: playback starts at 5 seconds and proceeds to 20 seconds whereupon it loops fro 15-20 seconds
7. Press the Loop button
8. Observe: subtle white lines disappear, meaning that Looping is inactive (though we still retain the Loop region)
9. Press Play
10. Observe: Playback starts at 5 seconds and stops at 10 seconds

**Actual behavior**
When Looping is active the visual cue (thin white lines around the button) are somewhat subtle.
There is no visual cue for the Play or Play-at-Speed button even though their behaviors a modified by the state change to looping

**Expected behavior**
Visual cue changes to the Play and Play-at-Speed buttons when looping is active - I would suggest te same icons that we used in the old shift-modified looping.

**Screenshots**
### Looping active - Loop button down
![image](https://user-images.githubusercontent.com/50205984/162730413-68e2e927-0f5e-4248-b456-63ca8ff7d3b2.png)

### Looping inactive - Loop button up (but note we retain the Loop region - _intended behavior_)
![image](https://user-images.githubusercontent.com/50205984/162730480-c1a1ba54-9e40-4dde-b6e3-a22ac0804305.png)

###Old Looping state change visual cues
![image](https://user-images.githubusercontent.com/50205984/162730557-6e089254-67d9-4ee4-9c7c-5fa341b267b8.png)

**Additional information (please complete the following information):**
 - OS: W10 but assume all OS
 - Version: Audacity 3.1.3 and 3.2.0 alpha master

**Additional context**
I have marked this issue as a regression as in earlier Audacity versions we had a **_much_** clearer visual cue (on the Play buttons) to indicate that Play would be Loop-Play.

Flagging @Tantacrul as a UX issue

**Comment by SteveDaulton** [comment]
I don't think that this is only a matter of visibility. The loop button has surprising / unexpected features. For example:

1. Generate a 30 second tone
2. Create a loop region from 10 to 15 seconds
3. Click the Loop button to turn looping off.
4. Select from 20 to 25 seconds
5. Play
6. The selected region plays once and then stops, as expected.
7. Click the Loop button to turn on looping
8. Play
9. Playback starts from the beginning of the selection. Play does not stop at the end of the selection but continues to the end of the track. Does not loop.

**Comment by LWinterberg** [comment]
> When Looping is active the visual cue (thin white lines around the button) are somewhat subtle.

this is the same visualization as the pressed state of the other buttons, is it not?

**Comment by SteveDaulton** [comment]
> > When Looping is active the visual cue (thin white lines around the button) are somewhat subtle.
> 
> this is the same visualization as the pressed state of the other buttons, is it not?

With the arguable exception of the Tools buttons, the Loop button is the only one that puts the app into a different [mode](https://en.wikipedia.org/wiki/Mode_(user_interface)), thus making it susceptible to mode errors.

**Comment by petersampsonaudacity** [comment]
It's interesting to compare the buttons in older Audacies where the older round buttons had a much clearer depressed/down visual cue:
![image](https://user-images.githubusercontent.com/50205984/162973536-20728134-6697-49a8-9e63-1e22c49e56ff.png)


And loop play (Shift modified play) also had a clear (2does what it says on the tin") visual cue
![image](https://user-images.githubusercontent.com/50205984/162973936-20a7a355-fcaf-44fb-b296-a45bd85d8474.png)

**Comment by LWinterberg** [comment]
This issue will get resolved with the 4.0 design, in combination with [a new timeline UX](https://wiki.audacityteam.org/wiki/GSoC_Ideas_2022#Audio_scrubbing_.26_timeline_improvements), which will allow us to do a better UX for the looping region itself (It's put out as a GSoC project but we'll do anyway later as time permits if nobody takes it) as well as #2101. 

The focus until then is to clear the beartraps which are more difficult to escape than pressing space when your playback continues to play when it shouldn't
